src/librustc_lexer/src/lib.rs

```diff
@@ -594,37 +656,49 @@ impl Cursor<'_> {
         false
     }
 
-    /// Eats the double-quoted string and returns a tuple of
-    /// (amount of the '#' symbols, raw string started, raw string terminated)
-    fn raw_double_quoted_string(&mut self) -> (usize, bool, bool) {
+    /// Eats the double-quoted string an UnvalidatedRawStr
```

Comment:
```suggestion
    /// Eats the double-quoted string an `UnvalidatedRawStr`.
```
Nit & general note: prefer backticks for type names and punctuate. :)